tests/ui/index.js

```diff
@@ -0,0 +1,59 @@
+var helpers = require('../lib/helpers');
+
+function testHomepage(test) {
+    test.assertTitle('Firefox Marketplace Style Guide');
+    test.assertVisible('.wordmark');
+    test.assertVisible('#page');
+}
+
+function testHamburger(test) {
+    var nav = '#site-header nav';
+    test.assertVisible('.hamburger');
+    test.assertNotVisible(nav);
+    casper.click('.hamburger');
+    test.assertVisible(nav);
+}
+
+casper.test.begin('Test mobile homepage', {
+    test: function(test) {
+        helpers.startCasper({path: '/'});
+        helpers.changeViewportMobile(test);
+
+        helpers.waitForPageLoaded(function() {
+            testHomepage(test)
+            test.assertNotVisible('#site-footer');
+            testHamburger(test)
+        });
+
+        helpers.done(test);
+    }
+});
+
+casper.test.begin('Test mobile homepage', {
+    test: function(test) {
+        helpers.startCasper({path: '/'});
+        helpers.changeViewportTablet(test);
```

Comment:
you can instead wrap the test object with helpers.tabletTest which sets up proper setUp/tearDown